packages/components/table/src/table-body/events-helper.ts

```diff
@@ -8,6 +8,10 @@ import type { TableColumnCtx } from '../table-column/defaults'
 import type { TableBodyProps } from './defaults'
 import type { TableOverflowTooltipOptions } from '../util'
 
+function compareValueWithError(a: number, b: number, error = 0.01) {
```

Comment:
I would suggest using a clearer name like `isGreaterThan` and a more common name for the `error` parameter like `epsilon`.

Reply:
Good suggestion, I've made changes.